winpkg/include/ddev.nsh

```diff
@@ -0,0 +1,112 @@
+/**
+ * ddev.nsh - LogicLib extensions for handling DDEV checks and actions
+ *
+ * The following "expressions" are available:
+ *
+ * DdevIsInstalled checks if ddev.exe is available in the provided directory
+ *
+ *   ${If} ${DdevIsInstalled} "$PROGRAMFILES64\DDEV"

```

Comment:
If I'm not mistaken, we're testing for the *directory* existence but then saying that the *executable* is there if we find the directory.

Reply:
This does check if the ddev.exe is found in the provided directory, ddev.exe is appended in the test but the directory must be provided because it is changable in the installer